Add test for bridge flags for enabling and disabling flooding on ports.

cappe987 opened the pull request:
Tests that flooding behaves as intended when both enabled and disabled, for both unicast and multicast.

Minor correction of a comment.

wkz commented on lib.sh:
```diff
@@ -111,7 +111,7 @@ ipaddr()
 #   g: global dest addr from arg (multicast)
 #   i: source addr from interface arg
-#   i: dest addr from interface arg
+#   I: dest addr from interface arg
```
Good catch! Please split it out in a separate commit :+1:

wkz commented on suite/20-basic.sh:
```diff
@@ -91,3 +91,49 @@ basic_learning_port()
+		report $h3 | grep -q "from $h1" || fail
+
+		step "Verify multicast flood on"
+		# Craft multicast address
+		mcast_da="\001\000\136\252\252\252" # 01:00:5e:aa:aa:aa
+		mcast_sa="$(addrfmt $(rndaddr))"
+		et="$(be16 0xbbbb)"
+		mcast_target="${mcast_da}${mcast_sa}${et}"
```
Any particular reason for generating a DA in the IPv4 multicast range? I think the SA could also just be `h1`s interface address. If yes, then the next comment applies :smile:

wkz commented on suite/20-basic.sh:
```diff
@@ -91,3 +91,49 @@ basic_learning_port()
+
+		bridge link set dev $b2 mcast_flood on
+		capture $h2 $h3
+		printf "$mcast_target" | { cat; echo from $h1; } | inject $h1
```
So I think this could just be:

`eth -g 1 -i $h1 |  { cat; echo from $h1; } | inject $h1`

wkz commented on suite/20-basic.sh:
```diff
@@ -91,3 +91,49 @@ basic_learning_port()
+		step "Verify multicast flood off"
+		bridge link set dev $b2 mcast_flood off
+		capture $h2 $h3
+		printf $mcast_target | { cat; echo from $h1; } | inject $h1
```
...and naturally the same applies here.

wkz reviewed (commented):
Great stuff! :1st_place_medal: 

Just want to see if we can avoid the escape-character-fest around the multicast part.

cappe987 commented:
No particular reason for the IPv4 multicast. I needed a multicast address and I made one :). Now I've switched to your suggestion. Also split up the correction into another commit.
